.github/workflows/npm-publish.yml

```diff
@@ -36,8 +36,15 @@ jobs:
       with:
         ssm_parameter: /devops/VA_VSP_BOT_NPM_TOKEN
         env_variable_name: VA_VSP_BOT_NPM_TOKEN
-    
-    - name: Publish build artifacts to NPM
-      run: yarn npm publish --access public --tolerate-republish
+
+    - name: Get Github Token password
+      uses: marvinpinto/action-inject-ssm-secrets@v1.2.1
+      with:
+        ssm_parameter: /devops/VA_VSP_BOT_GITHUB_TOKEN
+        env_variable_name: VA_VSP_BOT_GITHUB_TOKEN
```

Comment:
Where do these variables come from?

Reply:
They are pulled from the secret store in AWS so you use the AWS login action above and then you have access to those secrets